src/main/kotlin/org/jitsi/nlj/dtls/DtlsStack.kt

```diff
@@ -120,6 +120,9 @@ class DtlsStack(
 
     /**
      * A buffer we'll use to receive data from [dtlsTransport].
+     * TODO: we leak this buffer (it's never returned to the pool).  We can return
```

Comment:
Why do we care about leaking a single instance?

The pool is optimized for the request sizes we see most often, so it will just allocate java memory when we ask for 1500:
https://github.com/jitsi/jitsi-videobridge/blob/master/src/main/java/org/jitsi/videobridge/util/ByteBufferPool.java#L163

Reply:
Well I think we should strive to leak 0 packets, generally.  Are you saying don't use the pool here at all?  I'd agree that's probably the better way to go.

Reply:
Yes, I agree not using the pool is a better option